### ISSUE TITLE
remove ICP number from default template

### ISSUE THREAD

**Issue by breeswish** [vijos/vj4#339]

**Comment by masnn** [comment]
You can edit [builtin.py](https://github.com/vijos/vj4/blob/master/vj4/model/builtin.py#L331).

**Comment by twd2** [comment]
Yes, we know how to do, but we need a better solution to show the ICP number for our vijos.org, which is under-discussion.

**Comment by masnn** [comment]
Maybe you can try the environment variable.

**Comment by breeswish** [comment]
+1 for environment variable. Or we can use configurations.
